packages/collector/esm-register.mjs

```diff
@@ -20,6 +20,5 @@
 import instana from './src/index.js';
 instana();
 
-// We plan to utilize this for adding native ESM support in the near future
-// import { register } from 'node:module';
-// register(./loader.mjs, import.meta.url);
+import { register } from 'node:module';
+register('./loader.mjs', import.meta.url);
```

Comment:
Move loader.mjs into core
-> in core it will use its own core import-in-the-middle pkg

And here:

`require('@instana/core/loader.mjs')`